aws-ssm-patchbaseline/overrides.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
is this needed

Reply:
removed the override.json file was removed